src/.nuget/Microsoft.Build.Mono.NoSymbols.nuspec

```diff
@@ -0,0 +1,65 @@
+    <file src="$bin$/MSBuild.exe" target="lib" />
+    <file src="$bin$/MSBuild.exe.config" target="lib" />
+    <file src="$bin$/Microsoft.Common.CurrentVersion.targets" target="lib" />
+    <file src="$bin$/Microsoft.Common.overridetasks" target="lib" />
+    <file src="$bin$/Microsoft.Common.props" target="lib" />
+    <file src="$bin$/Microsoft.Common.targets" target="lib" />
+    <file src="$bin$/Microsoft.Common.tasks" target="lib" />
+    <file src="$bin$/Microsoft.CSharp.CurrentVersion.targets" target="lib" />
+    <file src="$bin$/Microsoft.CSharp.targets" target="lib" />
+    <file src="$bin$/Microsoft.Data.Entity.Shim.targets" target="lib" />
+    <file src="$bin$/Microsoft.NETFramework.CurrentVersion.props" target="lib" />
+    <file src="$bin$/Microsoft.NETFramework.CurrentVersion.targets" target="lib" />
+    <file src="$bin$/Microsoft.NETFramework.props" target="lib" />
+    <file src="$bin$/Microsoft.NETFramework.targets" target="lib" />
+    <file src="$bin$/Microsoft.ServiceModel.Shim.targets" target="lib" />
+    <file src="$bin$/Microsoft.VisualBasic.CurrentVersion.targets" target="lib" />
+    <file src="$bin$/Microsoft.VisualBasic.targets" target="lib" />
+    <file src="$bin$/Microsoft.VisualStudioVersion.v11.Common.props" target="lib" />
+    <file src="$bin$/Microsoft.VisualStudioVersion.v12.Common.props" target="lib" />
+    <file src="$bin$/Microsoft.VisualStudioVersion.v14.Common.props" target="lib" />
+    <file src="$bin$/Microsoft.WinFx.Shim.targets" target="lib" />
+    <file src="$bin$/Microsoft.WorkflowBuildExtensions.Shim.targets" target="lib" />
+    <file src="$bin$/Microsoft.Xaml.Shim.targets" target="lib" />
+    <file src="$bin$/Workflow.Shim.Targets" target="lib" />
+    <file src="$bin$/Workflow.VisualBasic.Shim.Targets" target="lib" />
+    <file src="$bin$/tools/*" target="lib/tools" />
+    <file src="$src$/Framework/**/*.cs" exclude="$src$/Framework/UnitTests/**/*.cs" target="src/Framework"/>
+    <file src="$src$/Shared/**/*.cs" exclude="$src$/Shared/UnitTests/**/*.cs" target="src/Shared" />
+    <file src="$src$/Utilities/**/*.cs" exclude="$src$/Utilities/UnitTests/**/*.cs" target="src/Utilities" />
+    <file src="$src$/XMakeBuildEngine/**/*.cs" exclude="$src$/XMakeBuildEngine/AllEngine/**/*.cs;$src$/XMakeBuildEngine/AllEngineAndConversion/**/*.cs;$src$/XMakeBuildEngine/allmajorprojects/**/*.cs;$src$/XMakeBuildEngine/ApexTests/**/*.cs;$src$/XMakeBuildEngine/SharedUtilities/**/*.cs;$src$/XMakeBuildEngine/UnitTests/**/*.cs;$src$/XMakeBuildEngine/UnitTestsPublicOM/**/*.cs" target="src/XMakeBuildEngine" />
+    <file src="$src$/XMakeCommandLine/*.cs" target="src/XMakeCommandLine" />
+    <file src="$src$/XMakeTasks/**/*.cs" exclude="$src$/XMakeCommandLine/DataDriven/**/*.cs;$src$/XMakeCommandLine/XamlRules/**/*.cs;$src$/XMakeCommandLine/DataDriven/**/*.cs;$src$/XMakeCommandLine/XMakeTasksUnitTests/**/*.cs" target="src/XMakeTasks" />
```

Comment:
The tools as the destination does not work. Either because of the reference items in the nuspec file or because I use the -symbols flag. That's the flag that creates a special symbols package on Windows with the sources and pdb files for debugging (Microsoft.Build.Debug.14.1.0.0-prerelease.symbols.nupkg). The Microsoft.Build.Debug.14.1.0.0-prerelease.nupkg includes only binaries. On Linux the Microsoft.Build.Mono.Debug.14.1.0.0-prerelease.nupkg (main and only) package contains sources in it. The symbols are not present for now on Linux since Roslyn does not create them.